shared/utils/combine-ovals.py

```diff
@@ -271,6 +271,20 @@ def check_is_loaded(loaded_dict, filename, version):
     return False
 
 
+def check_oval_version_from_oval(xml_content, oval_version):
+    oval_file_tree = ElementTree.fromstring(_header("", "") + xml_content + footer)
+    for defgroup in oval_file_tree.findall("./{%s}def-group" % oval_ns):
+        file_oval_version = defgroup.get("oval_version")
+
+    if file_oval_version is None:
```

Comment:
I think it's more future proof and readable to say that default OVAL version is 5.10 and just set it here. It will be the same thing in the end, just easier to verify and read.